### PR TITLE
Updating to latest Jackson that utilizes Istio Ingress Gateway routing. 

### PR DESCRIPTION
**Pull request by mtarng**

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -87,43 +81,12 @@ TBD - Create Keyvault solution with various secrets and env vars:
 - Various AAD Security env vars
 - UI env vars (if we get dynamic injection supported)
 
-### Deploy load balancer services
-`kubectl apply -f ui-service.yaml && kubectl apply -f ui-service.yaml`
-This step is currently needed so we can hardcode the `api` IP and expose the `UI` layer.
-
-### Create docker images
```
Comment (dtzar) [inline demo/README.md]: We should keep this for the UI images and tell people to deploy their own due to the limitation of client-side code baking in the variables at build time.

File: demo/1-ui-deployment.yaml [hunk 2]
```diff
@@ -13,8 +13,8 @@ spec:
         app: spring-boot-ui
     spec:
       containers:
-      - name: spring-boot-app-ui
-        image: <private docker repo>/ui:v2.0
+      - name: jackson-ui
+        image: mtarng/jackson-ui:v1.0
```
Comment (dtzar) [inline demo/1-ui-deployment.yaml]: Put in comment for people to replace this with their own UI image which points to their own cluster API endpoint

Comment (mtarng) [reply demo/1-ui-deployment.yaml]: With these changes, now the serves links that are relative paths. The routing is all handled through ingress gateway.

Comment (dtzar) [reply demo/1-ui-deployment.yaml]: How does the UI code resolve the address for `WEBPACK_PROP_API_BASE_URL` which is unique to the AKS cluster?

Comment (dtzar) [reply demo/1-ui-deployment.yaml]: Ah sorry - single public IP endpoint on Istio which serves up the UI and the APIs behind it.

Comment (mtarng) [reply demo/1-ui-deployment.yaml]: Yup. So we should only use the gateway. I removed the old load balancer services for api and the ui.

**Pull request merged**